chirp/drivers/kguv980p.py

```diff
@@ -2352,7 +2455,10 @@ def _decode(lst):
 
         for i in range(1, 11):
             x = str(i)
-            _str = _decode(eval("_settings.scanname"+x))
+            if self.MODEL != "KG-UV950P":
+                _str = _decode(eval("_settings.scanname"+x))
+            else:
+                _str = _decode_950(eval("_settings.scanname"+x))
```

Comment:
Here again, this should be just `self._decode()` and override it on the subclasses that need it.